### DISCUSSION TITLE
How to horizontally scale worker's containers.

### DISCUSSION THREAD

**Discussion by Amanraj0007** [airbytehq/airbyte#31665] [Connector Questions] [answered]
### What method are you using to run Airbyte?

Docker

### Platform Version or Helm Chart Version

latest

### What step the error happened?

On deploy

### Revelant information

I want to deploy airbyte on AWS ECS (docker deployment)
I want to horizontally scale the worker's container as load increases 
how could i do that.

**Comment by Amanraj0007** [comment]
@marcosmarxm

**Comment by marcosmarxm** [comment, marked as answer]
As described [here](https://github.com/airbytehq/airbyte/issues/4847) ECS is not officially supported.

[a comment by an automated account is not shown]
